java/src/main/native/src/row_conversion.cu

```diff
@@ -2005,7 +2174,8 @@ std::unique_ptr<table> convert_from_rows(lists_column_view const &input,
       });
 
   dim3 blocks(util::div_rounding_up_unsafe(gpu_tile_infos.size(), NUM_TILES_PER_KERNEL_FROM_ROWS));
-  dim3 threads(std::min(std::min(256, shmem_limit_per_tile / 8), static_cast<int>(child.size())));
+  dim3 threads(
+      std::min(std::min(NUM_THREADS, shmem_limit_per_tile / 8), static_cast<int>(child.size())));
```

Comment:
What's the 8 for here?

Reply:
It felt reasonable to have each thread read/write 8 bytes of data. This is a somewhat arbitrary selection that could, and probably should, be tweaked in the performance pass.

Reply:
Maybe
`constexpr auto bytes_copied_per_thread = 8`?
or something along those lines.  Lot of buffers, indices and offsets being slung around here.  The more clarity on what everything is doing the better.

Reply:
Yeah, I agree. Lots of `std::min`s here and it seems like it should be a constant at the top of the file.